### DISCUSSION TITLE
Gherkin upgrade

### DISCUSSION THREAD

**Discussion by KonstantinosArch** [reqnroll/Reqnroll#204] [General]
Pretty simple question. Gherkin has left almost 10 versions behind (19.0.3 and the latest is 28.0.0). Is there any plan for an upgrade to get the latest features?

**Comment by Code-Grump** [comment]
Are there any new additions to the Gherkin language that you would expect to be supported?

**Reply by gasparnagy** [reply]
The changelog is [here](https://github.com/cucumber/gherkin/blob/main/CHANGELOG.md). 

What is related to us is a couple of keyword translation updates and that there is a new "keyword type" field in the step (that we also calculated for ourselves, now this would not be required, I guess).

I think it would be good to upgrade. Let's see if I will have time for it today. If not, anyone can pick it up.

**Reply by gasparnagy** [reply]
on the way... https://github.com/reqnroll/Reqnroll/pull/205
